src/java.base/share/classes/java/util/zip/ZipInputStream.java

```diff
@@ -172,13 +173,14 @@ public void closeEntry() throws IOException {
     }
 
     /**
-     * Returns 0 after EOF has reached for the current entry data,
-     * otherwise always return 1.
+     * Returns 0 when end of stream is detected for the current ZIP entry or
+     * {@link #closeEntry()} has been called on the current ZIP entry, otherwise always return 1.
      * <p>
```

Comment:
The updated wording looks okay. One small thing is that I think it would be a bit better if the word "always" were dropped.

Reply:
> The updated wording looks okay. One small thing is that I think it would be a bit better if the word "always" were dropped.

Removed always and just pushed another commit with that change.  Thank you for the feedback